data/projects.ini

```diff
@@ -814,7 +814,6 @@ build_dependencies: fuse,libcrypto,zlib
 dpkg_build_dependencies: libfuse-dev,libssl-dev,zlib1g-dev
 msi_name: libewf-python
 pkg_configure_options: --enable-python,--with-pyprefix
-version: >=20160318
```

Comment:
Is this EWF change intended? Please change the PR name if so

Reply:
It is, solves an issue with minimum check not picking up on the latest legacy version

Reply:
BTW you could have just changed this on merge seeing the commit is named "Changes for build of requests and libewf projects"